docs/configuration/configuration.md

```diff
@@ -0,0 +1,136 @@
+
+```powershell
+# Invoke with a different product name
+Invoke-SCuBA `
+  -ConfigFilePath typical_config.yaml `
+  -ProductNames defender
+```
+
+### Full Use
+
+The [full config file](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Sample-Config-Files/full_config.yaml) shows all of the global parameters supported by ScubaConfig specified in the config file. Any one of these parameters may be commented out. If not specified or if commented out, ScubaConfig will supply the default value, unless it's overridden on the command line. Default values do not apply to authentication parameters.
+
+```powershell
+# Invoke without any overrides
+Invoke-SCuBA -ConfigFilePath full_config.yaml
+```
+
+## Generate an Empty Sample Configuration File
+
+ScubaGear's support module can generate an empty sample config file. Running the `New-Config` cmdlet will generate a full sample config called `SampleConfig.yaml` that can be filled out based on the guidance below. Parameters can be passed to the `New-Config` cmdlet to change values inside the sample config.
+
+```powershell
+# Create an empty config file
+New-Config
+```
+
+## Product-specific Configuration
+
+Config files can include a top-level level key for a given product whose values are related to that specific product. For example, look for the value of `Defender` in this [Defender config file](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Sample-Config-Files/defender-config.yaml). Currently, only Entra ID and Defender use this extra configuration.
+
+Under a product key, there can be policy keys that provide configuration values unique to the product. In the Defender config file, for example, there is the `MS.DEFENDER.1.4v1` key.
+
+## Configuring Entra ID Policy Exclusions
+The ScubaGear configuration file provides the capability to exclude specific users or groups from some of the Entra ID policy checks. For example, a user could exclude emergency access accounts from some of the policy checks. Exclusions must only be used if they are approved within an organization's security risk acceptance process. **Exclusions can introduce grave risks to your system and must be managed carefully**.
+
+An example configuration file for Entra ID can be found in [this sample config file](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Sample-Config-Files/aad-config.yaml).
+
+### Conditional Access Policy Exclusions
+
+The `Aad` top level key in the configuration allows the user to specify configurations specific to the Entra Id baseline. Under the `Aad` key is the policy identifier such as `MS.AAD.1.1v1` and under that is the `CapExclusions` key where the excluded users or groups are defined. The `CapExclusions` key supports both a Users or Groups list with each entry representing the UUID of a user or group from the tenant that will be excluded from the respective policy check. Below is an example screenshot that shows these items.
```

Comment:
If we're adding backticks for `Aad`, `MS.AAD.1.1v1`, `CapExclusions`, etc. we should add them for `Users` and `Groups` keys as well.

Reply:
Added.